Global-API/health.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import FastAPI
+from sqlalchemy import create_engine, exc
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+app = FastAPI()
+
+DATABASE_URL = "TBD"
```

Comment:
Let's load this from a .env file that is not added to the repository (since it will contain a password among other secrets, make sure to add it to a .gitignore file).

You can then have a local version of this file to connect to your own database for testing, and the server will have its own as well.